doc/help/pypeit_show_2dspec.rst

```diff
@@ -0,0 +1,25 @@
+.. code-block:: console
```

Comment:
Are these being generated on-the-fly?  Just checking.

Reply:
Yep.